ykrt/src/jitstats.rs

```diff
@@ -1,6 +1,21 @@
+/// This module records statistics about Yk and the VM it is part of. The accuracy of the
+/// statistics varies: in general, the statistics about yk's internals are accurate, but
+/// interactions with the wider interpreter are likely to be approximations, because we can only
+/// guess at what the interpreter is doing. For example, when an interpreter spins up a thread,
+/// does that count as "time spent interpreting"? What happens if it's a job queue that immediately
```

Comment:
Why don't we renaming the interpreting state to `NotInYk` (or similar), then it's obvious we aint talking about the interpreter.

Reply:
How about `OutsideYk`?

Reply:
See 90a9858 as an example.